main/src/ca/uwaterloo/flix/language/phase/typer/ConstraintGen.scala

```diff
@@ -681,10 +681,15 @@ object ConstraintGen {
         val resEff = Type.Pure
         (resTpe, resEff)
 
-      case Expr.Without(exp, _, _) =>
-        // We ignore the `without` here.
-        // TODO EFF-MIGRATION Use set subtraction when we have set effects.
+      case Expr.Without(exp, effSymUse, _) =>
+        //
+        // e: tpe \ eff - effSymUse
```

Comment:
Why does it say effSymUse here? It should just be effSym right?

Reply:
it is a `Symbol.EffSymUse`

Reply:
oh you mean in the typing rule, yes that's true